sources/world/admin_divisions_v1.hjson

```diff
@@ -0,0 +1,171 @@
+{
+    versions: '1'
+    type: http
+    production: true
+    wikidata: Q56061
+    note: 'Administrative divisions of countries collected from Natural Earth and OpenStreetMap'
+    data: ''
+    attribution: [
+      {
+        label: {
+          en: Made with NaturalEarth
+        }
+        url: {
+          en: http://www.naturalearthdata.com/about/terms-of-use
+        }
+      }
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
```

Comment:
The field-names/descriptions are kind of hard to wrap your head around when you see them displayed:

e.g.: The field-descriptions looks like in the Maps-UX

![image](https://user-images.githubusercontent.com/1833023/86928571-b6d92180-c102-11ea-9b0f-e3e1934edd9e.png)


If we ingest this file into Kibana Maps (e.g. using GeoJson upload), and use the raw documents, we see the field-names.

![image](https://user-images.githubusercontent.com/1833023/86928763-fe5fad80-c102-11ea-86f6-421d2d652fd5.png)

I'd consider renaming and using the ECS names for these fields.

```
 {
          type: id
          name: region_iso_code
          desc: iso2 region code
}
{
          type: id
          name: country_iso_code
          desc: iso2 country code
}
```


[EDIT] nevermind, I see that we already made some changes. Just wondering if we should be using the ECS-names.

Reply:
Field names and descriptions now match the ECS names.